_cucumber/cucumber.rake

```diff
@@ -27,7 +27,17 @@ task :features do |t, args|
     profile = 'parallel'
   end
 
-  exit_status = runner.run(profile)
+  if ENV['CUCUMBER_TAGS'].to_s.empty?
+    tags = nil
+  else
+    tags = ENV['CUCUMBER_TAGS']
+    known_good_cucumber_tags = %w(@wip @smoke @cdk_beta @ignore)
+    unless known_good_cucumber_tags.include?(tags)
+      raise("#{tags} is not a known tag. If required it can be added to the known good cucumber tags in #{File.dirname(__FILE__)}/cucumber.rake!")
+    end
+  end
+
+  exit_status = runner.run(profile, tags)
   p "exit status was #{exit_status}"
```

Comment:
Cool. As I see this now, only one tag can be passed in. Otherwise  known_good_cucumber_tags.include?(tags) would be strange things. As such I think we should rename CUCUMBER_TAGS to CUCUMBER_TAG